src/main/play-doc/operation/ConsolidatedLogging.md

```diff
@@ -40,6 +40,28 @@ The provided Elasticsearch bundle configuration is using these settings:
 
 To change the settings create a new bundle configuration by modiying the `bundle.conf` file inside of the bundle configuration zip file. The configuration file is named `elasticsearch-prod-{digest}.zip` and is located in the `extra` folder as well. Afterwards reload the bundle with the new configuration.
 
+### Elasticsearch bundle rolling upgrade
+
+Elasticsearch stores its data within the filesystem. As such, to preserve existing data when performing deployment, it's important to ensure the data is replicated to the new instance of Elasticsearch.
```

Comment:
when performing deployment
maybe
when moving running instances
or
when managing the deployment